src/cljs/nr/gameboard.cljs

```diff
@@ -2234,7 +2232,7 @@
                                  #(card-highlight-mouse-over % value button-channel)
                                  :on-mouse-out
                                  #(card-highlight-mouse-out % value button-channel)
-                                 :id {:code value}}
+                                 :id (:title value)}
```

Comment:
Intentional change from a map to a function call?

Reply:
Yep. I honestly don't know how it was working before because it was setting the `id` string value in the HTML to `[object Object]` lol.